CMakeLists.txt

```diff
@@ -63,6 +63,14 @@ if(BUILD_WITH_OPENMP)
     endif()
 endif()
 
+option(BUILD_EASYLOGGINGPP "Build EasyLogging++ as a part of the build" ON)
+
+if (BUILD_EASYLOGGINGPP)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBUILD_EASYLOGGINGPP=1")
```

Comment:
Isn't it better to use add_definitions for preprocessor definitions?

Reply:
Good idea. I will do that. I am still learning this cmake stuff.